index.js

```diff
@@ -134,9 +134,29 @@ document.querySelectorAll('.btnModal').forEach((n) => n.addEventListener('click'
 
 // =============validtion form=============
 
-const email = document.getElementById('emailForm');
 const gontactForm = document.getElementById('contact-form');
 const messageform = document.querySelector('.messageform');
+const email = document.getElementById('emailForm');
+const name = document.getElementById('name');
+const msg = document.getElementById('msg');
+
+const dataForm = JSON.parse(localStorage.info) || [];
+
+gontactForm.addEventListener('change', (e) => {
+  e.preventDefault();
+  email.classList.remove('invalide');
+  messageform.classList.remove('invalide');
+  messageform.style.display = 'none';
+  // localStorage
+  const newDataForm = {
+    name: name.value,
+    email: email.value,
+    msg: msg.value,
+  };
+  dataForm.push(newDataForm);
+  localStorage.setItem('info', JSON.stringify(dataForm));
```

Comment:
- [ ] Kindly store an `object` and not an `array` in the `localStorage`. You don't need the `dataForm` array. 👍 

#### Hint: `localStorage.setItem('info', JSON.stringify(newDataForm))`